python/cuml/linear_model/linear_regression.pyx

```diff
@@ -228,8 +229,8 @@ class LinearRegression(Base,
         }[algorithm]
 
     @generate_docstring()
-    def fit(self, X, y, convert_dtype=True,
-            sample_weight=None) -> "LinearRegression":
+    def fit_(self, X, y, convert_dtype=True,
+             sample_weight=None) -> "LinearRegression":
```

Comment:
We should document the convention of naming of the internal methods, and `_fit` would be better to follow python conventions